engine/install/rhel.md

```diff
@@ -49,7 +49,8 @@ $ sudo yum remove docker \
                   docker-logrotate \
                   docker-engine \
                   podman \
-                  runc
+                  docker-compose-plugin
+                  runc \
```

Comment:
I don't think this one is needed (if it's installed, it'd be one of our packages; this section is to uninstall "other" packages, or packages that are deprecated). The trailing `\` should also be after the plugin, not after `runc` 😅 

```suggestion
                  runc
```